features/appmgt/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/themes/appm/js/publisher.commons.js

```diff
@@ -266,7 +293,14 @@ $(document).ready(function() {
 	function checkNonSpecial(value){
         	var non_special_regex = /^[A-Za-z][A-Za-z0-9\s-]*$/;
         	return non_special_regex.test(value);
-    	}
+    }
+
+    function validateImages(files){
+        var inputType = files[0].type;
```

Comment:
Fixed in https://github.com/wso2/carbon-appmgt/pull/539/commits/3fb4b6fbca128ac480fbd3dfb4881b29f7dc570e